ground/src/main/proto/com/google/ground/loi.proto

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.google.android.ground.proto";
+option java_multiple_files = true;
+
+import "com/google/ground/geometry.proto";
+import "com/google/ground/audit_info.proto";
+
+// A location of interest (LOI, presented to the user as "[data collection]
+// sites"). All data submitted by data collectors is associated with an LOI.
+message LocationOfInterest {
+    // Required. A system-defined ID.
+    string id = 1;
+
+    // Required. The system-defined of the job to which this LOI belongs.
+    string job_id = 2;
+
+    // Required. The geometry associated with this LOI.
+    Geometry geometry = 3;
+
+    // Required. The number of submissions associated with this LOI.
+    int32 submission_count = 4;
+
+    // Required. The user ID of the LOI's owner.
+    string owner_uid = 5;
+
+    // Required. Info about who created this LOI and when.
+    AuditInfo created = 6;
+
+    // Info about who last modified this LOI and when. If unset, it is assumed
+    // the LOI was never modified after it was created.
+    AuditInfo last_modified = 7;
+
+    // A user-defined ID for this location of interest.
```

Comment:
Maybe "name" or "description" instead, to connote human readability? I wouldn't want it being used accidentally as a key. Ok to keep as is if that's what we have right now.

Reply:
This is used to store the `fid` / `feature_id` / `system:index` exported by different systems, as opposed to `name` which is a GeoJSON property. Which reminds me, I need to add those as well! I could also call it "feature_id", but we don't have the concept of "feature" anywhere anymore, so that might be confusing. Open to suggestions.

Reply:
How about something like `custom_tag`? or `tag`?